packages/mobile/screens/SignUp.tsx

```diff
@@ -0,0 +1,244 @@
+            }
+            <Input>
+              <Field
+                component={TextInputCust}
+                value={username}
+                placeholder="Name..."
+                placeholderTextColor="#003f5c"
+                error={touched["username"] && errors["username"]?.length > 0}
+                onBlur={handleBlur('username')}
+                onChangeText={handleChange('username')} />
+                
+            </Input>
+
+            {touched.email && errors?.email?.length &&
+              <Text style={{ fontSize: 10, color: 'red' }}>{errors.email}</Text>
+            }
+            <Input>
+              <Field
+                component={TextInputCust}
+                value={email}
+                placeholder="Email..."
+                placeholderTextColor="#003f5c"
+                error={touched["email"] && errors["email"]?.length > 0}
+                onBlur={handleBlur('email')}
+                onChangeText={handleChange('email')} />
+            </Input>
+
+            {touched.password && errors?.password?.length &&
+              <Text style={{ fontSize: 10, color: 'red' }}>{errors.password}</Text>
+            }
+            <Input>
+              <Field
+                component={TextInputCust}
+                value={password}
+                secureTextEntry
+                placeholder="Password..."
+                placeholderTextColor="#003f5c"
+                error={touched["password"] && errors["password"]?.length > 0 }
+                onBlur={handleBlur('password')}
+                onChangeText={handleChange('password')} />
+            </Input>
+
+            {touched.confirmation && errors?.confirmation?.length &&
+              <Text style={{ fontSize: 10, color: 'red' }}>{errors.confirmation}</Text>
+            }
+            <Input>
+              <Field
+                component={TextInputCust}
+                value={confirmation}
+                secureTextEntry
+                placeholder="Confirmation..."
+                placeholderTextColor="#003f5c"
+                error={touched["confirmation"] && errors["confirmation"]?.length > 0}
+                onBlur={handleBlur('confirmation')}
+                onChangeText={handleChange('confirmation')} />
+            </Input>
+
+
+            {errors?.server &&
+              <ErrorText>{errors.server}</ErrorText>
+            } 
+            <SignUpButton
+              onPress={handleSubmit}>
+              <SignUpText>SIGN UP</SignUpText>
+            </SignUpButton>
+          </React.Fragment>
+        )}
+      </Formik>
+      <SignInFrame>
+        <Text>{'Already have an acount? '}</Text>
+
+        <TouchableOpacity onPress={() => navigation.navigate("SignIn")}>
+          <LoginText> LOGIN</LoginText>
+        </TouchableOpacity>
+      </SignInFrame>
+    </Container>
+  );
+}
+
+const Header = styled.View`
+  margin-bottom: 25px;
+  `;
+ 
+const TextInputCust = styled(TextInput)`
```

Comment:
Seems like you're reusing a lot of components on both views, why not export them in a view and import them on the other one if they're the same one?

Reply:
later

Reply:
is only copy paste. And could have advantage not reusing.